src/autoscaler/operator/appmetricsdb_test.go

```diff
@@ -39,13 +40,14 @@ var _ = Describe("AppMetricsDB Prune", func() {
 
 	Describe("Prune", func() {
 		JustBeforeEach(func() {
-			appMetricsDbPruner.Operate()
+			appMetricsDbPruner.Operate(context2.TODO())
```

Comment:
Since logging becomes somewhat more important now (trace IDs are being logged to correlate logs and traces), shall we enhance the tests to assure that trace ids are being logged?

Reply:
That's a good idea! For now, I have removed the logging related changes from this PR, see below.